### PR TITLE
add vagrantfile to test CI and fixes

### PR DESCRIPTION
**Pull request by rphillips**

### CODE REVIEW

**Review by saschagrunert — APPROVED** [review]
LGTM

**Comment by saschagrunert** [comment]
@rphillips can you please dco sign-off the commits?

File: Vagrantfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ Vagrant.configure("2") do |config|
 
     cd /vagrant
     echo "localhost" >> hosts
+    ansible-galaxy install -r contrib/test/ci/requirements.yml
```
Comment (sohankunkerkar) [inline Vagrantfile:28]: hmm... I think we don't have `contrib/test/ci/requirements.yml`

Comment (rphillips) [reply Vagrantfile:32]: added the file

**Comment by TomSweeneyRedHat** [comment]
LGTM
assuming happy tests

**Comment by rphillips** [comment]
ci-e2e, ci-integration, ci-critest all passed

**Pull request merged**